README.md

```diff
@@ -533,6 +571,63 @@ project.getMetadata(function(err, metadata) {
 ```
 
 
+## Google Cloud Speech (Limited Preview)
+
+> **This is a Limited Preview release of Google Cloud Speech.** This feature is not covered by any SLA or deprecation policy and may be subject to backward-incompatible changes.
+
+- [API Documentation][gcloud-speech-docs]
+- [Official Documentation][cloud-speech-docs]
+
+```js
+var gcloud = require('gcloud');
+
+// Authorizing on a per-API-basis. You don't need to do this if you auth on a
+// global basis (see Authorization section above).
+
+var speech = gcloud.speech({
+  projectId: 'my-project',
+  keyFilename: '/path/to/keyfile.json'
+});
+
+// Detect the speech in an audio file.
+speech.detect('./audio.raw', {
+  encoding: 'LINEAR16',
+  sampleRate: 16000
+}, function(err, result) {
+  // result = {
+  //   transcript: 'how old is the Brooklyn Bridge',
+  //   confidence: 0.97234234234
+  // ]
+});
+```
+
+```js
+// Detect the speech in an audio file stream.
+var initialRequest = {
+  encoding: 'LINEAR16',
+  sampleRate: 16000,
+  continuous: true
+};
+
+var stream = speech.recognize(initialRequest);
+  .on('data', function (recognizeResponse) {
+    console.log(recognizeResponse);
+  });
+
+stream.write({
+  audioRequest: {
```

Comment:
Ping me when that happens or open an issue on google-proto-files so I can
publish the update.
On Thu, Jul 7, 2016 at 5:41 PM Jason Dobry <notifications@github.com> wrote:

> In README.md
> <https://github.com/GoogleCloudPlatform/gcloud-node/pull/1407#discussion_r69992828>
> :
>
> > +
> > +```js
> > +// Detect the speech in an audio file stream.
> > +var initialRequest = {
> > +  encoding: 'LINEAR16',
> > +  sampleRate: 16000,
> > +  continuous: true
> > +};
> > +
> > +var stream = speech.recognize(initialRequest);
> > +  .on('data', function (recognizeResponse) {
> > +    console.log(recognizeResponse);
> > +  });
> > +
> > +stream.write({
> > +  audioRequest: {
>
> Yes, my initial implementation of Speech support for gcloud-node uses the
> v1 proto. As soon as the v1beta1 proto is published to
> https://github.com/googleapis/googleapis (and thus available in the
> google-proto-files NPM package), I will update this PR to reflect the
> changes in the new proto.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-node/pull/1407/files/162b0683a27ae1f4f49eba986d1a19d13aedf526#r69992828>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAsIaMs9aG7rPCr0KKCTSxonyYCHg7oeks5qTXKTgaJpZM4JDmJK>
> .
>